website/docs/index.html.markdown

```diff
@@ -365,7 +365,7 @@ for more information about connecting to alternate AWS endpoints or AWS compatib
 
 The `assume_role` configuration block supports the following optional arguments:
 
-* `duration_seconds` - (Optional) Number of seconds to restrict the assume role session duration.
+* `duration_seconds` - (Optional) Number of seconds to restrict the assume role session duration. Must be set to a value of at least 900.
```

Comment:
Hi @anuraaga! Thanks for this clarification, while we are here maybe we can also add a note to clarify the upperbound?

```suggestion
* `duration_seconds` - (Optional) Number of seconds to restrict the assume role session duration. You can provide a value from 900 seconds (15 minutes) up to the maximum session duration setting for the role.
```

Reply:
Thanks for the suggestion!